container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1835,19 +1823,19 @@ configuration:
     description: "The database role tag of autoscaler postgres database"
   - name: AUTOSCALER_DEFAULT_BREACH_DURATION_SECS
     default: 300
-    description: "The default breach duration for eventgenerator of autoscaler"
+    description: "The default breach duration for eventgenerator of autoscaler, in seconds."
   - name: AUTOSCALER_DEFAULT_COOLDOWN_SECS
     default: 300
-    description: "The default cooldown seconds between two scaling action for eventgenerator of autoscaler"              
+    description: "The default cooldown duration between two scaling action for eventgenerator of autoscaler, in seconds."              
   - name: AUTOSCALER_DEFAULT_STAT_WINDOW_SECS
     default: 300
-    description: "The default statistic window seconds between two scaling action for eventgenerator of autoscaler"   
+    description: "The default statistic window duration between two scaling action for eventgenerator of autoscaler, in seconds."   
   - name: AUTOSCALER_EVENT_GENERATOR_AGGREGATOR_APP_METRIC_CHANNEL_SIZE
     default: 1000
-    description: "The size of golang channel used by autoscaler eventgenerator.metricpoller as a buffer to save metrics by batch"
+    description: "The size of golang channel used by autoscaler eventgenerator.metricpoller as a buffer to save metrics by batch."
   - name: AUTOSCALER_EVENT_GENERATOR_AGGREGATOR_APP_MONITOR_CHANNEL_SIZE
     default: 200
-    description: "The size of golang channel used by autoscaler eventgenerator.aggregator and eventgenerator.metricpoller to exchange data"
+    description: "The size of golang channel used by autoscaler eventgenerator.aggregator and eventgenerator.metricpoller to exchange data."
   - name: AUTOSCALER_EVENT_GENERATOR_AGGREGATOR_EXECUTE_INTERVAL
     default: "40s"
     description: "The duration defines how long does autoscaler eventgenerator aggregate appmetrics"
```

Comment:
`The time interval used by the Autoscaler Event Generator to aggregate app metrics.`

Reply:
There are 8 variables with no description below.
`AUTOSCALER_EVENT_GENERATOR_LOGGING_LEVEL` needs to be removed. [`LOG_LEVEL`](https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/role-manifest.yml#L2620) must be used instead.